spec/controllers/vms_controller_spec.rb

```diff
@@ -101,6 +108,32 @@
     end
   end
 
+  describe 'PATCH #update' do
+    before do
+      sign_in admin
+      allow(vm1).to receive(:users=)
```

Comment:
The allow(vm).to receive calls might be more useful in the general vm mock (spec/api/v_sphere_api_mocker.rb method: vim_vm_mock)